src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -270,8 +288,24 @@ export default class PhotoField extends React.Component {
     }
   }
 
+  cropImage = () => {
+    const cropResult = {};
+    this.refs.cropper.getCroppedCanvas().toBlob(blob => {
+      const file = blob;
+      file.lastModifiedDate = new Date();
+      file.name = this.state.fileName;
```

Comment:
The file constructor isn't supported in IE or Edge, is this format acceptable, or do we need to use a [polyfill](https://github.com/moxiecode/moxie) to create the file to send back? @jbalboni @lihanli